webhook-service/handler/handler.go

```diff
@@ -196,10 +198,10 @@ func (th *TaskHandler) performWebhookRequests(webhook lib.Webhook, eventAdapter
 	executedRequests := 0
 	logger.Infof("executing webhooks for subscriptionID %s", webhook.SubscriptionID)
 	for _, req := range webhook.Requests {
-		request, err := CreateRequest(req)
+		request, err := th.CreateRequest(req)
 		if err != nil {
-			logger.Infof("creating CURL request failed: %s", err.Error())
-			return nil, err
+			logger.Errorf("creating CURL request failed: %s", err.Error())
```

Comment:
here we might to check for specific error types, as e.g. the detection of a disallowed URL is not an error in the execution of the webhook service, so that should probably be logged on another error level

Reply:
So do you suggest have Info there ?